clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/AutoScalingWorker.groovy

```diff
@@ -267,6 +269,13 @@ class AutoScalingWorker {
       throw ex
     }
 
+    if (lifecycleHooks != null && lifecycleHooks.size() > 0) {
```

Comment:
```suggestion
    if (lifecycleHooks != null && !lifecycleHooks.isEmpty()) {
```

Prefer `isEmpty` over `size` for this case.

Reply:
Done @robzienert, thanks for the suggestion.